guides/_subsections/automated-analysis.md

```diff
@@ -52,30 +52,6 @@ Cryostat also provides an *Automated Analysis Card* that is able to display the
     %}
   </li>
   <li>
-    {% capture download-report-text %}
-    <p>
-      <figure>
-        <a href="{{ site.url }}/images/2.4.0/automated-analysis-4.png" target="_blank">
-          <img src="{{ site.url }}/images/2.4.0/automated-analysis-4.png">
-        </a>
-        <figcaption>
-          View the report on its own without connecting to Cryostat.
-          Check <i>Show OK Results</i> to include results where the rules
-          engine found no issues in the recording.
-        </figcaption>
-      </figure>
-      <figure>
-        <a href="{{ site.url }}/images/2.4.0/automated-analysis-5.png" target="_blank">
-          <img src="{{ site.url }}/images/2.4.0/automated-analysis-5.png">
-        </a>
-        <figcaption>
-          To download the HTML <i>Automated Analysis</i> report to local disk, right click 
-          the page and select <i>Save Page As...</i>. Alternatively, press <kbd>Ctrl</kbd>+<kbd>S</kbd> 
-          on Windows/Linux, or <kbd>⌘</kbd>+<kbd>S</kbd> on macOS.
-        </figcaption>
-      </figure>
-    </p>
-    {% endcapture %}
```

Comment:
Could you undo this change and put it into the next PR for updating the written content? Just so that the final upstream PRs that will be merged into the release branch are as cleanly separated as possible and any rebase/merge conflicts are as easy to understand as possible.

Reply:
This didn't quite get undone - it looks like this has the copy from `main` instead of the copy from the base branch, or something like that.